.github/workflows/python-publish.yml

```diff
@@ -1,18 +1,12 @@
 name: Python Publish to PYPI
 on:
-  # Automatic release based on either creating a published release or
-  # publishing a draft release (creating a draft release will not trigger the
-  # "release" event type according to the docs).
-  # The workflow file version used is that from the tag corresponding to the
-  # release. This is why the "workflow_dispatch" trigger is configured as well:
-  # in case we need to manually re-publish a tag using a modified workflow
   release:
-    types: [created, published]
```

Comment:
decided Python publish is safe enough now to just trigger automatically after a Release PR merge